src/Periphery/ServiceFeeCollector.sol

```diff
@@ -40,36 +40,6 @@ contract ServiceFeeCollector is TransferrableOwnership {
 
     /// External Methods ///
 
-    /// @notice Collects gas fees
-    /// @param tokenAddress The address of the token to collect
-    /// @param feeAmount The amount of fees to collect
-    /// @param chainId The chain id of the destination chain
-    /// @param receiver The address to send gas to on the destination chain
-    function collectTokenGasFees(
-        address tokenAddress,
-        uint256 feeAmount,
-        uint256 chainId,
-        address receiver
-    ) external {
-        LibAsset.depositAsset(tokenAddress, feeAmount);
-        emit GasFeesCollected(tokenAddress, chainId, receiver, feeAmount);
-    }
-
-    /// @notice Collects gas fees in native token
-    /// @param chainId The chain id of the destination chain
-    /// @param receiver The address to send gas to on destination chain
-    function collectNativeGasFees(
-        uint256 chainId,
-        address receiver
-    ) external payable {
-        emit GasFeesCollected(
-            LibAsset.NULL_ADDRESS,
-            chainId,
-            receiver,
-            msg.value
-        );
-    }
-
     /// @notice Collects insurance fees
```

Comment:
Does `collectNativeInsuranceFees` not need to be updated as well to accept an input amount instead of handling `msg.value`?